iscsi/iscsi.go

```diff
@@ -23,12 +23,16 @@ import (
 
 // iscsi target started with /sys/kernel/config/target/iscsi/iqn*
 // configfs + target/iscsi/iqn*
-// IqnGlob is representing all the possible IQN
-const IqnGlob = "target/iscsi/iqn*"
+// iqnGlob is representing all the possible IQN
+const iqnGlob = "target/iscsi/iqn*"
 
-// TargetCore static path /sys/kernel/config/target/core for node_exporter
+// targetCore static path /sys/kernel/config/target/core for node_exporter
 // reading runtime status
-const TargetCore = "target/core"
+const targetCore = "target/core"
+
+// devicePath static path /sys/devices/rbd/[0-9]* for rbd devices to
+// read at runtime status
+const devicePath = "../../devices/rbd"
```

Comment:
Should this one be relative to sysfs (`/sys`) instead of configfs (`/sys/kernel/config`)?  A user could potentially have sysfs mounted to a different location, and that would break this relative path.

Reply:
Yes I agree. However the hard part is how to "remember" the sys path and configfs path for fixture test.  Because we only setup path Once in the NewFS, so I either use relative path for /sys or relative path for "/sys/kernel/config". Or I can update NewFS construct with two different path /sys and kernel/config with two variables.